migrate-to-hugo.md

```diff
@@ -0,0 +1,88 @@
+## Website Elements
+
+- Title
+- Description
+- Tabs that switches between categories
+  - Apps
+  - Tools (difference from apps?)
+  - Archives
+  - Videos
+  - Articles
+  - Discussions
+- Below, after clicking category, show most popular tag within that category
+  and allow further filtering. (future allow multiple tags? Will be hard without JS)
```

Comment:
I want to clarify the intention here: we want to be able to assign multiple tags to an item, right?
Otherwise its just a "subcategory".

Reply:
Yes, absolutely, I should have added that in the example data files. Items should be able to have multiple tags, but on the website itself it'll be hard to generate static pages for every combination, but the items themselves should indeed have multiple tags.

Reply:
Is that even possible without JS?

Reply:
@hacdias it's not impossible :) But it'll generate **a lot** of pages if we're gonna generate every single combination, so let's not do that. But at least we should generate listing pages for each label (singular)